src/test/scala/io/citrine/lolo/transformers/FeatureRotatorTest.scala

```diff
@@ -80,6 +80,22 @@ class FeatureRotatorTest {
     }
   }
 
+  /**
+    * Functions getLoss and getPredictedVsActual should pass through to base learner.
+    */
+  @Test
+  def testPassthroughFunctions(): Unit = {
+    val rotatedTrainingResult = FeatureRotator(GuessTheMeanLearner()).train(data)
```

Comment:
`GuessTheMeanLearner` is probably too trivial of a case, here, since the predicted labels will be uniform.  I'd recommend a single decision tree (since its still fast).

Reply:
Fair enough. Done.